helm/lib/cortex-core/values.yaml

```diff
@@ -0,0 +1,282 @@
+# Copyright 2025 SAP SE
+# SPDX-License-Identifier: Apache-2.0
+
+owner-info:
+  helm-chart-url: "https://github.com/cobaltcore-dev/cortex/helm/lib/cortex"
+  maintainers:
+    - "p.matthes@sap.com"
+    - "markus.wieland@sap.com"
+    - "arno.uhlig@sap.com"
+  support-group: "cobaltcore-dev"
+  enabled: true
+
+image:
+  repository: ghcr.io/cobaltcore-dev/cortex
```

Comment:
I'd factor this out to
```
global:
  imageRegistry: ghcr.io

image:
  repository: cobaltcore-dev/cortex
```

The registry can than easily be overwritten even from umbrella charts.